infra/identity/prod/westeurope/main.tf

```diff
@@ -19,6 +19,56 @@ provider "azurerm" {
   }
 }
 
-data "azurerm_subscription" "current" {}
+resource "azurerm_resource_group" "rg_identity" {
+  name     = "${local.project}-identity-rg"
+  location = local.location
+
+  tags = local.tags
+}
+
+module "federated_identities" {
+  source = "../../../modules/azure_identity_federation_github"
+
+  prefix       = "dx"
+  env_short    = "p"
+  env          = "prod"
```

Comment:
can we have these values taken from a local file to the whole prod/westeurope directory?

Reply:
Do you mean the locals.tf file? Yes we can